pages/playground.tsx

```diff
@@ -8,9 +8,10 @@ import { Container } from 'components/ui'
 const PlaygroundPage = () => {
   return (
     <>
+      <html lang="en"></html>
       <Head>
         <meta property="og:type" content="website" />
-        <meta property="title" content="EVM Codes - Playground" />
+        <title>EVM Codes - Playground</title>
```

Comment:
You don't want to add something here EVM Codes - Explore your contracts in our Playground" or something?

Reply:
That'd be too long – the longest tagline we'd want is for the homepage. This particular text is what you'd see in a tab bar, the shorter, the better.